pkg/sql/logictest/testdata/logic_test/drop_index

```diff
@@ -105,14 +105,8 @@ GRANT CREATE ON TABLE users TO testuser
 
 user testuser
 
-statement error in use as unique constraint
-DROP INDEX users@bar
-
-statement error in use as unique constraint
-DROP INDEX users@bar RESTRICT
-
 statement ok
-DROP INDEX users@bar CASCADE
```

Comment:
is there an implicit case in which we have a test for?
do we have a test for someone changing the column type to `unique` making this implicit?